bot/auction/commands.py

```diff
@@ -37,7 +38,8 @@ def update_files(self, data):
             f.write(json.dumps(self.history, indent=4))
 
     @commands.Cog.listener()
-    async def on_message(self, msg):
+    async def on_message(self, msg,):
```

Comment:
Tutaj się przecinek po msg zakręcił, trzeba usunąć

Reply:
> Tutaj się przecinek po msg zakręcił, trzeba usunąć

To coś zmieniło?

Reply:
brzydko wygląda ;)